command/clean/clean.js

```diff
@@ -1,5 +1,7 @@
 'use strict';
 
+const { tryDelete } = require("../../lib/discordjs-utils");
```

Comment:
I feel like `tryDelete` is a strange name not really relevant. 
What about `deleteChannelMesssage` or something like this so we know it only delete messages from channels and not dm ?

Reply:
I agree with that
i was thinking about the fact that it try to delete, if it works the message will be deleted otherwise it will stay in the channel
Before renaming the function we need to try it in a NewsChannel

Reply:
Not matter where the command is typed the bot should try to delete it and ignore if it fails.
Should we write something in the logs when the deletion fail ?

Reply:
We will have to much logs if we do that 
I will do some tests